certbot-nginx/tests/boulder-integration.conf.sh

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
In fact, it is required to make the Nginx integration tests run correctly. Indeed previously, the same FQDN `nginx.wtf`, was used to generate a certificate using the HTTP challenge, then to generate a certificate using the TLS challenge.

However, with this new functionnality, the TLS challenge will not be processed, as the HTTP challenge is still valid. Consequently, the Nginx integration tests are failing because of a drop on the code coverage.

Using a different FQDN, we ensure that the TLS challenge is effectively processed, and so that the coverage of the tests is correct.

Reply:
Makes sense. Thanks for the explanation!